### ISSUE TITLE
Incorrect typing of $(this:...) function

### ISSUE THREAD

**Issue by vitalyiegorov** [webdriverio/webdriverio#6772]
**Describe the bug**
You cannot create correct typings for custom wrappers like `testID$(selector:string, context: WebdriverIO.Browser | WebdriverIO.Element)` and use them through `browser.addCommand(...)`.

**To Reproduce**
Example `testID` helper function:
```
export const testID$ = async (
    testID: string,
    context: WebdriverIO.Browser | WebdriverIO.Element = browser
): Promise<WebdriverIO.Element> => await context.$(`~${testID}`);
```

See https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/commands/browser/%24.ts#L61

It should have same types as 
https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/commands/browser/%24%24.ts#L53

**Expected behavior**
`$` to have same type for `this` as `$$`

**Comment by christian-bromann** [comment]
I don't think this is the correct way of solving this. We should consolidate the commands similar how we did this with `$$`. It seems that the implementations for `$` for the browser as well as the element are the same.

**Comment by christian-bromann** [comment]
fixed in #6810
